src/components/SchemaTree.tsx

```diff
@@ -42,10 +44,24 @@ export const SchemaTree: React.FC<ISchemaTree> = props => {
 
   return (
     <TreeList
+      className="sl-flex-1"
       draggable={false}
-      striped
+      interactive={false}
```

Comment:
Are you sure about this one?
There is no pointer cursor when you hover over a row.

Reply:
True, that was to remove the hover background but probably can be done differently.

Reply:
Actually interactive mode looks good now, so might stay ;)